tools/validators/instance_validator/instance_parser.py

```diff
@@ -114,12 +114,23 @@ def parse_yaml(filename):
     translation = yaml[top_name][_TRANSLATION]
     translation.revalidate(_TRANSLATION_SCHEMA)
 
-    # TODO can this be automatically verified based on ontology?
```

Comment:
I dont think so, @berkoben ?

Reply:
technically it might be possible to validate if a translation that someone specified manually was UDMI compliant, but in that case specifying it manually isn't strictly an error.  If they say it is, there's no way to tell it's not without seeing the telemetry

Reply:
ok I see